test/unit/simulator-specs.js

```diff
@@ -12,17 +12,23 @@ import B from 'bluebird';
 import xcode from 'appium-xcode';
 
 
-let should = chai.should();
+chai.should();
 chai.use(chaiAsPromised);
 
+const UDID = 'C09B34E5-7DCB-442E-B79C-AB6BC0357417';
```

Comment:
it's from the 'devices.json' file that we import. so yes will always be there. I can add a comment explaining where it came from.
Hm, actually, ill just get it from the `devices` object. nicer that way anyways.